tests/tests.py

```diff
@@ -19,18 +19,22 @@ def test_readme():
 
 def test_pipeline(tmp_path):
     tmp_model_path = tmp_path / "models"
-
+    tmp_results_path = tmp_path / "results" / "source_data"
+    tmp_results_path.parent.mkdir()
     copytree("models", tmp_model_path)
+    copytree("results/source_data", tmp_results_path)
 
     # run pipeline
     cmd = shlex.split("bash run --nostata --nocensus --num-proj 2")
     subprocess.run(cmd, check=True)
 
     bad_files = []
     len_path = len(tmp_path.parts)
-    for p in tmp_model_path.rglob("*"):
+    for p in list(tmp_model_path.rglob("*.csv")) + list(
+        tmp_results_path.rglob("*.csv")
```

Comment:
@bolliger32 there is one Excel spreadsheet, `results/source_data/ExtendedDataTable3.xlsx` so we should either ask everyone to output CSV files or just change this code to grab either file types.

Reply:
@luna983 pointed out that its easier for us to leave this and leave it in xlsx form. Since it's not programmatically generated, I think its fine to not include it in the tests